tests/dvslib/dvs_acl.py

```diff
@@ -136,6 +136,16 @@ def create_acl_rule(self, table_name, rule_name, qualifiers, action="FORWARD", p
 
         self.config_db.create_entry("ACL_RULE", "{}|{}".format(table_name, rule_name), fvs)
 
+    def create_mirror_acl_rule(self, table_name, rule_name, qualifiers, priority="2020"):
```

Comment:
This is a more generic version of create_acl_rule. What I would do is make this method take a mirror stage and then automatically add the MIRROR_ACTION qualifier.

What I want to do next is make this generic method into a helper and then provide `create_l3_acl_rule` (which will replace the existing `create_acl_rule` method) and `create_mirror_acl_rule`, which will both use this helper under the hood. However, this change would touch a lot of files so I'm not going to ask you to do that here. :)